docs/resources/test.md

```diff
@@ -213,11 +219,13 @@ resource "kentik-synthetics_test" "minimal-dns-test" {
     ]
```

Comment:
`dns` and `dns-grid` tests **must** have **only** `dns` in the `tasks` list. Those tests do not support running `ping` and `traceroute`.